dotnetv3/dynamodb/FromSQL/ListItemsExample/ListItemsExample/ListItems.cs

```diff
@@ -45,25 +49,25 @@ static void Main()
             }
             else
             {
-                Console.WriteLine("Could not find " + configfile);
+                Console.WriteLine($"Could not find {configfile}");
                 return;
             }
 
             var empty = false;
             var sb = new StringBuilder("You must supply a non-empty ");
```

Comment:
Is this complete - 'You must supply a non empty " ?

Reply:
The contents of the string are concatenated over the following lines. The string is only printed if a parameter is missing.